retroshare.pri

```diff
@@ -150,37 +150,37 @@ rs_jsonapi:CONFIG -= no_rs_jsonapi
 # To enable forums indexing append the following assignation to qmake command
 # line "CONFIG+=rs_deep_forums_index"
 CONFIG *= no_rs_deep_forums_index
-rs_deep_forums_index:CONFIG -= no_rs_deep_forums_index
+rs_deep_forums_index:DEFINES *= RS_DEEP_FORUMS_INDEX
```

Comment:
This and the other ones like this one is incorrect. 
also was the previous version blocking compilation??

Reply:
i try with old values and it's ok. i will rollback